library/common/src/main/java/com/google/android/exoplayer2/util/NalUnitUtil.java

```diff
@@ -252,17 +292,16 @@ public static int getH265NalUnitType(byte[] data, int offset) {
   }
 
   /**
-   * Parses an SPS NAL unit using the syntax defined in ITU-T Recommendation H.264 (2013) subsection
-   * 7.3.2.1.1.
+   * Parses a SPS NAL unit payload using the syntax defined in ITU-T Recommendation H.264 (2013)
+   * subsection 7.3.2.1.1.
    *
    * @param nalData A buffer containing escaped SPS data.
-   * @param nalOffset The offset of the NAL unit header in {@code nalData}.
+   * @param nalOffset The offset of the NAL unit in {@code nalData}.
```

Comment:
Can we add "payload" into this sentence? (maybe even explicitly clarify what it does not include).